rust/datafusion/src/execution/context.rs

```diff
@@ -651,13 +662,19 @@ impl ExecutionConfig {
         self.create_default_catalog_and_schema = create;
         self
     }
+
+    /// Enables the `information_schema` virtual tables
+    pub fn with_information_schema(mut self) -> Self {
+        self.information_schema = true;
```

Comment:
Can it be disabled as well - or don't we expect that?

Reply:
Would it be worth enabling by default, even, and adding an option to _disable_ it here? I can't think of any downsides to enabling it, and for the subset of the info schema that we support, it's in line with the standard (as far as I'm aware without spending stupid money of a copy of the standard 😅)

Reply:
> Can it be disabled as well - or don't we expect that?

I didn't expect people to want to be able to change the setting at runtime on the same `Context` -- either they would want the information schema or not. However, for consistency with the other settings I think it would be good to allow the flag to be passed in and I will do so

> Would it be worth enabling by default, even, and adding an option to disable it here? 

There might be some small runtime overhead for always enabling it. I don't have any strong preference on one way or the other. I am happy to change the default. Does anyone else have any thoughts?